### DISCUSSION TITLE
how to import word file？

### DISCUSSION THREAD

**Discussion by moyuwang110** [requarks/wiki#6219] [Help / Questions] [answered]
how to import word file

**Comment by fnafox** [comment, marked as answer]
If you mean import the document as a page, that is not possible to my knowledge. However what you could do is use something like word2md to turn your word document into markdown. Alternatively, copy pasting from word to the visual editor should maintain a fair bit of formatting. If maintaining exact formatting is a priority, then uploading it as a pdf and embedding the document inside the wiki page is probably your best option.

**Comment by jackadam1981** [comment]
I use the multi doc converter to convert Word documents into HTML files.

And I wrote a Python script to download the original text and place it at the top of the page.

Of course, we are still working hard to modify CSS. Although HTML files can be indexed, CSS stylesheets cannot be effective. We need to write HTML stylesheets into Wiki.js. We are still trying automation solutions.



This content can still be edited on Wiki.js in the future.
